type-definitions/tests/.flowconfig

```diff
@@ -0,0 +1,12 @@
+[libs]
+../immutable.js.flow
+
+[ignore]
+.*/node_modules/.*
+
+[options]
+suppress_comment=^\\( \\|\n\\)*\\$ExpectError\\(([^)]*)\\)?$
```

Comment:
What is this line doing? Is there a way to comment the .flowconfig to explain this?